agents/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh

```diff
@@ -98,6 +98,7 @@ elif echo ${LINUX_TYPE} | grep -qi "debian"; then
 
     if [ ${INSTALL_JDK} = true ]; then
         PACKAGES=${PACKAGES}" openjdk-8-jdk-headless";
+        echo "deb http://ftp.de.debian.org/debian jessie-backports main" | ${SUDO} tee --append /etc/apt/sources.list
```

Comment:
I used that url in a comment as an example. Yes, we should use this one I think. Or install by adding ppa.